desktop/cmp/grid/Grid.js

```diff
@@ -304,6 +305,7 @@ class Grid extends Component {
     }
 
     onSelectionChanged = (ev) => {
+        if (!this.model.agNodesRendered) return;
```

Comment:
Curious about the return here. Would this fire when the nodes were not rendered? What would be the general state of the grid in such a case? Would it throw, or would the API return something strange / misleading for `getSelectedRows()`?

I ask because otherwise it seems like it would be better to still call `selModel.select()` and let the selModel update its record collection. The reaction above would be short-circuited, but presumably also be a no-op in that the grid itself triggered the selection event - and in any case it would ensure all in sync on the next go round...